asmcli/lib/installation_dependencies.sh

```diff
@@ -66,20 +67,6 @@ EOF
      ;;
     *);;
   esac
-}
-
-# [START required_apis]
-required_apis() {
-  local CA; CA="$(context_get-option "CA")"
-    cat << EOF
-mesh.googleapis.com
-EOF
-  case "${CA}" in
-   gcp_cas)
-     echo privateca.googleapis.com
-     ;;
-    *);;
-  esac
```

Comment:
same here. Is there a reason we delete these lines?

Reply:
Same reason as above.